core-bundle/src/Resources/contao/config/config.php

```diff
@@ -386,7 +385,6 @@
 	'regular'   => PageRegular::class,
 	'forward'   => PageForward::class,
 	'redirect'  => PageRedirect::class,
-	'root'      => PageRoot::class,
```

Comment:
How can I add root pages in the site structure if you remove this?

Reply:
https://github.com/contao/contao/blob/5.x/core-bundle/src/Controller/Page/RootPageController.php